docker_daemon/check.py

```diff
@@ -446,6 +446,9 @@ def _get_tags(self, entity=None, tag_type=None):
                                 tags.append("kube_replication_controller:%s" % replication_controller)
                                 tags.append("pod_name:%s" % pod_name)
 
+                        elif k == KubeUtil.CONTAINER_NAME_LABEL and Platform.is_k8s():
+                            if v:
+                                tags.append("pod_container_name:%s" %v)
```

Comment:
i think `kubernetes_container_name` would be clearer? I think containers are called apps in rkt, but I fear `pod` is not kubernetes-specific enough and could confuse people.